src/themes/scripts/updateThemes.ts

```diff
@@ -29,8 +29,16 @@ import { generateTheme } from '../utils/generateTheme'
 
 const getFile = (theme: string, file: string): string => resolve(THEMES_DIR, theme, file)
 
+const folderNameToIgnore = [
```

Comment:
Why is there a .DS_Store in the theme directory?

Reply:
It is a folder inserted automatically by Finder of Mac OS. This is only a guard